### ISSUE TITLE
Not correctly render in mobile resolution

### ISSUE THREAD

**Issue by andersonalvesalmeida** [Zodiase/meteor-mdl#34]
# Not correctly render in mobile resolution

I am unable to make the resolution of my app to suit a mobile device, I created a meteor project and replace the file code mdltest.html by:

`````` html
<head>
  <title>mdltest</title>
</head>

<body>
  <!-- Always shows a header, even in smaller screens. -->
    <div class="mdl-layout mdl-js-layout mdl-layout--fixed-header">
      <header class="mdl-layout__header">
        <div class="mdl-layout__header-row">
          <!-- Title -->
          <span class="mdl-layout-title">Title</span>
          <!-- Add spacer, to align navigation to the right -->
          <div class="mdl-layout-spacer"></div>
          <!-- Navigation. We hide it in small screens. -->
          <nav class="mdl-navigation mdl-layout--large-screen-only">
            <a class="mdl-navigation__link" href="">Link</a>
            <a class="mdl-navigation__link" href="">Link</a>
            <a class="mdl-navigation__link" href="">Link</a>
            <a class="mdl-navigation__link" href="">Link</a>
          </nav>
        </div>
      </header>
      <div class="mdl-layout__drawer">
        <span class="mdl-layout-title">Title</span>
        <nav class="mdl-navigation">
          <a class="mdl-navigation__link" href="">Link</a>
          <a class="mdl-navigation__link" href="">Link</a>
          <a class="mdl-navigation__link" href="">Link</a>
          <a class="mdl-navigation__link" href="">Link</a>
        </nav>
      </div>
      <main class="mdl-layout__content">
        <div class="page-content">My content goes here</div>
      </main>
    </div>
</body>```


When I open in the google chome, emulating a Galax S5, 360 x 640 px, the app is as desktop appearance, all small.

The same code without the meteor works correctly.

![image](https://cloud.githubusercontent.com/assets/6940620/14116962/c4103056-f5b8-11e5-977c-14bbf5d5741e.png)

What should I do?

Thanks.
``````

**Comment by Zodiase** [comment]
I don't know how you could get a "correct" result just with those code. Because with this codepen I couldn't get a mobile view on either chrome or the iOS simulator.
http://codepen.io/Zodiase/pen/KzXqVJ
![image](https://cloud.githubusercontent.com/assets/1815174/14214308/07239086-f802-11e5-9c17-454acb81dd8e.png)
![image](https://cloud.githubusercontent.com/assets/1815174/14214388/6931ea8e-f802-11e5-9d87-9c84fdde4ea7.png)

This issue really is about how your page's meta is set up.

In short, you need this in `<head>`:

``` HTML
<meta name="viewport" content="width=device-width, initial-scale=1">
```

And that has nothing to do with either Meteor or MDL.

**Comment by Zodiase** [comment]
With no follow-ups I assume this is resolved. Let me know if you still have this problem.
